### PR TITLE
modify readme and add a file for common terms

### PR DESCRIPTION
**Pull request by dantereve**
I've modified the readme and give some useful information in a cheerful tone. Some people might be afraid to contribute  I guess, so I hope it will help a bit. 

We will have to keep track of troubleshooting terms to translate and add them as they appear in common-terms .md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,39 @@
...
-(Feel free to translate this document and add any content you feel would be useful to contributors).
-  
\ No newline at end of file
+## Comment traduire une page ? 
+
+Plusieurs possibilités s'offrent à vous. 
+
+__Si vous n'avez jamais contribué à Gatsby__, faites un fork de ce répertoire. Vous pourrez ensuite éditer le fichier directement dans l'éditeur proposé par GitHub. 
+
+Une fois terminé, faites un _pull-request_. Votre traduction sera alors examinée par un ou plusieurs mainteneurs du projet qui pourront à ce moment-là proposer des modifications à apporter à votre traduction concernant l'orthographe, la syntaxe ou la terminologie qui peut parfois être complexe à traduire.
```
Comment (mindsers) [inline README.md]: I think pull request is feminine, isn’t it ?

File: common-terms.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Cette liste est évolutive et constitue une simple proposition afin de conserver une cohérence d'ensemble, n'hésitez pas à la modifier pour rajouter ou corriger une traduction. 
+
+Vous pouvez également venir en discuter librement dans le salon dédié `localization-fr` sur le [Discord](https://gatsby.dev/discord)  
+
+| Terminologie originale      | Traduction proposée               |
+|---------------------|---------------------------|
+| Array               | Array                     |
+| Build               | Générer / Génération      |
+| Hot-reloading       | Rechargement instantané ? |
```
Comment (mindsers) [inline common-terms.md]: If we don’t have a strong opinion about a term, maybe we should not putting it in this file. 

We could open an issue centralize the discussion or discuss it in discord. The advantage of the issue system is that it keeps things organized.

**Review by mindsers — CHANGES_REQUESTED** [review]
This is a great idea! Only few thing that might be changed but nothing big.

File: README.md (hunk 1 above)
Comment (dantereve) [reply README.md]: yep you're right

File: common-terms.md (hunk 2 above)
Comment (dantereve) [reply common-terms.md]: Keeping track of this in an issue might be a good idea! 💪

**Review by gatarelib — APPROVED** [review]

**Review by mindsers — APPROVED** [review]

**Pull request merged**